tfjs-node/README.md

```diff
@@ -66,6 +66,49 @@ To use this package on Raspberry Pi, you need to rebuild the node native addon w
 $ npm rebuild @tensorflow/tfjs-node --build-from-source
 ```
 
+#### Custom binaries URI
+
+If you happen to be using a mirror for the libtensorflow binaries (default is [https://storage.googleapis.com/]), you have 3 options (in order of priority):
+
+1. Set the environment variable `CDN_STORAGE`
+
+```sh
+CDN_STORAGE="https://yourmirrorofchoice.com" npm install <package>
+(or)
+CDN_STORAGE="https://yourmirrorofchoice.com" yarn install <package>
+```
+
+2. Set the environment variable `TFJS_NODE_CDN_STORAGE`. This has the same behavior as `CDN_STORAGE`, but introduced to prevent collisions with other npm packages that might use `CDN_STORAGE`.
```

Comment:
This is bit duplicated, maybe better to add deprecation note on the CDN_STORAGE, since TFJS_NODE_CDN_STORAGE seems to be a better naming.

Reply:
Yes, I agree. It would be great to deprecate the use of `CDN_STORAGE`. Is there an "official" way to do this other than to leave a note that `CDN_STORAGE` will be deprecated at some future date? I will do this now.

Reply:
You can put the option 2 and 3 at the top and move the CDN_STORAGE to the end and indicate it as a deprecated options.

Reply:
I've completed this. Thanks.